developer-docs/examples/bolt_spec_example/spec/example_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
```

Comment:
You should be able to run this from `developer-docs/examples/bolt_spec_examples` (make sure out spec docker containers are running) with `bundle exec rake spec`.

Reply:
Instead of going for breadth maybe depth is more important here showing how you can:
1. run a task or plan
2. run a command to check that the task did what it was supposed to